.changeset/real-balloons-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@propeldata/ui-kit': minor
+---
+
+Added SimpleFilter component and FilterProvider
```

Comment:
Is this format correct? I saw a different format in @hasancolak's PR (https://github.com/propeldata/ui-kit/pull/160). Can we standardize?

Reply:
I don't think we are using any prefix right now just starting with a verb (add, adjust, export, unlock)

![image](https://github.com/propeldata/ui-kit/assets/84721399/a476c244-3938-499b-a890-448ad44a96d7)